tracing/src/lib.rs

```diff
@@ -913,6 +906,8 @@ mod macros;
 
 pub mod dispatcher;
 pub mod field;
+/// Attach a span to a `std::future::Future`.
+pub mod instrument;
```

Comment:
IDK if I really think this needs its own module, I might either reexport it from the root and make the instrument module hidden, or put it in the `span` module. but, i don't really care.